test/e2e/ui-properties/randplanetBtn

```diff
@@ -0,0 +1,21 @@
+/**
+ * Created by ruxit on 5/19/16.
+ */
+
+module.exports = {
```

Comment:
I saw that there was the ui-locators.js file, and wasn't sure if I should create my own and possibly duplicate code, or build on what was already in the framework. I'll convert that to a page object.